src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -375,20 +500,51 @@ private void loadConfig(ContestConfig config) {
     textFieldUndervoteLabel.setText(config.getUndervoteLabel());
     textFieldUndeclaredWriteInLabel.setText(config.getUndeclaredWriteInLabel());
     textFieldRulesDescription.setText(config.getRulesDescription());
-    radioBatchEliminationTrue.setSelected(config.isBatchEliminationEnabled());
-    radioContinueUntilTwoCandidatesRemainTrue.setSelected(
-        config.willContinueUntilTwoCandidatesRemain());
-    radioExhaustOnDuplicateCandidateTrue.setSelected(config.isExhaustOnDuplicateCandidateEnabled());
-    radioTreatBlankAsUndeclaredWriteInTrue.setSelected(
-        config.isTreatBlankAsUndeclaredWriteInEnabled());
+    setToggleBoolean(
+        config.isBatchEliminationEnabled(), radioBatchEliminationTrue, radioBatchEliminationFalse);
+    setToggleBoolean(
+        config.willContinueUntilTwoCandidatesRemain(),
+        radioContinueUntilTwoCandidatesRemainTrue,
+        radioContinueUntilTwoCandidatesRemainFalse);
+    setToggleBoolean(
+        config.isExhaustOnDuplicateCandidateEnabled(),
+        radioExhaustOnDuplicateCandidateTrue,
+        radioExhaustOnDuplicateCandidateFalse);
+    setToggleBoolean(
+        config.isTreatBlankAsUndeclaredWriteInEnabled(),
+        radioTreatBlankAsUndeclaredWriteInTrue,
+        radioTreatBlankAsUndeclaredWriteInFalse);
+  }
+
+  // TODO: this function assumes that if a field isn't provided in the config, it'll be false
+  private void setToggleBoolean(boolean condition, RadioButton radioTrue, RadioButton radioFalse) {
```

Comment:
I wonder if it would be easier to represent these boolean params with checkboxes instead of radios.

Reply:
Checkboxes would allow you to select more than one option, so that's a no-go. The radio button toggle group only allows one option to be selected at a time (so will automatically unselect any others).